bitnami/influxdb/templates/relay/deployment.yaml

```diff
@@ -0,0 +1,97 @@
+{{- if eq .Values.architecture "high-availability" }}
```

Comment:
Nice. I didn't know you could put the templates into folders! Much better than doing `relay-deployment.yaml` or `relay-svc.yaml`.